components/tokens/custom-spectrum/custom-medium-vars.css

```diff
@@ -65,4 +65,14 @@ governing permissions and limitations under the License.
 
   --spectrum-alert-dialog-padding: var(--spectrum-spacing-500);
   --spectrum-alert-dialog-description-to-buttons: var(--spectrum-spacing-700);
+
+  --spectrum-steplist-step-width: 80px;
+  --spectrum-steplist-marker-diameter: 8px;
+  --spectrum-steplist-marker-hitArea: 20px;
+  --spectrum-steplist-segment-height: 2px;
+  --spectrum-steplist-topPadding: 22px;
+  --spectrum-steplist-small-topPadding: 11px;
+  --spectrum-steplist-sidePadding: 60px;
+  --spectrum-steplist-label-labelOffset: 10px;
+  --spectrum-steplist-label-text-size: 12px;
```

Comment:
If these don't change from medium to large, then they are fine to stay in the index.css file rather than moving them out here